src/ai-services/assistant-manager.service.ts

```diff
@@ -26,6 +28,7 @@ export class AssistantManagerService {
     this.assistants.set('dory', this.doryService);
     this.assistants.set('gpt-4', this.gpt4Service);
     this.assistants.set('bloom', this.bloomService);
+    this.assistants.set('doryPlus', this.DoryPlusService);
```

Comment:
you should keep the name consistent with other names, so this must be `dory-plus` and not `doryPlus`